cf_remote/remote.py

```diff
@@ -208,8 +209,17 @@ def get_info(host, *, users=None, connection=None):
     else:
         data["os"] = "unix"
 
-        scp("nt-discovery.sh", host, connection, hide=True)
-        discovery = parse_envfile(ssh_sudo(connection, "bash nt-discovery.sh"))
+        cf_remote_dir = dirname(__file__)
+        script_path = join(cf_remote_dir, "nt-discovery.sh")
+        stat(script_path)
+        result = scp(
+            script_path,
+            host,
+            connection,
+            hide=True,
+            rename="/tmp/nt-discovery.sh",
+        )
+        discovery = parse_envfile(ssh_sudo(connection, "bash /tmp/nt-discovery.sh"))
```

Comment:
@craigcomstock when doesn't ssh and scp agree on CWD?

Reply:
It's not an uncommon thing to have a `CWD` or even a chroot set for `scp`.

Reply:
It stemmed from using localhost which uses LocalConnection instead. I will double-check. The primary issue was that nt-discovery.sh wasn't included in the manifest of the package and so wasn't included in the pip install and so wasn't available except in CWD where you ran cf-remote.

The move to /tmp was a change I made because it bothered me in the localhost case that it just put nt-discovery.sh any old place that I might happen to be, dirtying up the directory. As I mentioned in another comment I will try to refactor to use something like `$HOME/.cfengine/cf-remote/tmp` or similiar. :+1: Thanks!

Reply:
I refactored to using HOME for local and reverted to "empty" aka $HOME or whatever ssh/scp defaults to for remote.